main.go

```diff
@@ -198,18 +193,36 @@ func main() {
 		klog.Fatalf("Leader election lost, shutting down")
 	}
 
-	go startLeaderElection(leClient, recorder, becameLeader, lostLeader)
+	go func() {
+		if err = startLeaderElection(leClient, recorder, becameLeader, lostLeader); err != nil {
+			fatal(cableEngineSyncer, "Error starting leader election: %v", err)
+		}
+	}()
+
 	<-stopCh
 	klog.Info("All controllers stopped or exited. Stopping main loop")
 }
 
+func submarinerClusteFrom(submSpec *types.SubmarinerSpecification) types.SubmarinerCluster {
```

Comment:
```suggestion
func submarinerClusterFrom(submSpec *types.SubmarinerSpecification) types.SubmarinerCluster {
```

Reply:
> Typo ;-).

I had just woke up and didn't have my readers on 😄